defaults/main.yml

```
@@ -1 +1,20 @@
 clamav_cron_frequency: weekly
+
+# A dictionary of values to be configured in the clamd and freshclam configuration files.
+# Multivalued keys are specified as lists.
+# A Nonetype value means that the key will be removed from the file.
+# Example:
+#
+# clamav_freshclam_conf:
+#   DatabaseMirror: ['db.local.clamav.net', 'database.clamav.net']
+#   Bytecode: 'true'
+#   PrivateMirror:
+# clamv_clamd_conf:
+#   LogFile: /var/log/clamav/clamav.log
+
+
+clamav_freshclam_conf: {}
+clamav_clamd_conf: {}
```

With my previous comments applied.

```suggestion
clamav_freshclam_configuration: {}
clamav_clamd_configuration: {}
```